src/main/scala/org/constellation/p2p/Download.scala

```diff
@@ -89,7 +88,24 @@ class SnapshotsProcessor(downloadSnapshot: (String, Iterable[APIClient]) => IO[S
       .flatMap(acceptSnapshot)
   }
 
+  private def downloadSnapshot(hash: String, pool: Array[EnhancedAPIClient]): IO[StoredSnapshot] = {
```

Comment:
Why it's needed here if `downloadSnapshot` is passed as an argument and should be abstracted from here?

Reply:
I don't quite know what you mean Marcin. This is really the same as before, I just renamed APIClient.

Reply:
Not really. In the changes I can see that you copied part of the code and pasted here. You can see it on the diff. And I think this is just some dummy mistake :) but there shouldn’t be this code

Reply:
Interesting. Maybe a merge issue? I don't remember copying this on purpose. But regardless, I'll get rid of it if it's not needed. Good catch.